README.md

```diff
@@ -1,3 +1,7 @@
 # wlgranados.github.io
 
-This repo is for my personal website.
+Hey, I'm William Granados!
+
+I'm a third year student in Computer Science and Statistics at the University of Toronto, My interests include: programming, pokemon, and video games.
```

Comment:
I have other interests now, including: tv shows (like brooklyn nine-nine, game of thrones,etc), dancing (bachata), and games (like mafia, werewolves, avalon, etc).